live_view/live_view_studio/lib/live_view_studio_web/router.ex

```diff
@@ -0,0 +1,58 @@
+defmodule LiveViewStudioWeb.Router do
+  use LiveViewStudioWeb, :router
+
+  pipeline :browser do
+    plug :accepts, ["html"]
+    plug :fetch_session
+    plug :fetch_live_flash
+    plug :put_root_layout, {LiveViewStudioWeb.LayoutView, :root}
+    plug :protect_from_forgery
+    plug :put_secure_browser_headers
+  end
+
+  pipeline :api do
+    plug :accepts, ["json"]
+  end
+
+  scope "/", LiveViewStudioWeb do
+    pipe_through :browser
+
+    get "/", PageController, :index
+
+    live "/light", LightLive
```

Comment:
@samu975 what is the difference between live and other routing options like a get or a post?

Reply:
Live gets routed to the LiveView module it means is only used in live view